flare/predict.py

```diff
@@ -0,0 +1,105 @@
+"""
+Helper functions which obtain forces and energies
+corresponding to atoms in structures
+"""
+from flare.env import AtomicEnvironment
+from flare.gp import GaussianProcess
+from flare.struc import Structure
+
+import numpy as np
+import concurrent.futures
+
```

Comment:
These methods are similar to the ones in otf.py, so this file duplicates a lot of code. Before modifying otf.py to call these functions, we'll need to check that they work properly on 32 core jobs (parallelization has been finicky in the past)

Reply:
Sounds good and I agree. We can consult @nw13slx  and @YuuuuXie ; I know at least Lixin has worked with an implementation similar to this on her branch.